libcaf_net/src/net/web_socket/framing.cpp

```diff
@@ -60,24 +61,27 @@ ptrdiff_t framing::consume(byte_span buffer, byte_span) {
     if (opcode_ == nil_code) {
       // Call upper layer.
       if (hdr.opcode == detail::rfc6455::connection_close) {
-        up_->abort(make_error(sec::connection_closed));
+        // TODO
```

Comment:
What's the todo item?

Reply:
I wanted to check the following: 
These two places don't have a message argument, so I can provide an empty message to keep the interface uniform, or keep the `make_error` outside instead of inside the `abort_and_close` function, or make the `abort_and_close` take an variadic template.